src/main/kotlin/de/docs_as_co/intellij/plugin/drawio/editor/DiagramsEditor.kt

```diff
@@ -54,9 +72,22 @@ class DiagramsEditor(private val project: Project, private val file: VirtualFile
                 }
             }
         }
+
     }
 
-    private fun saveFile(data : ByteArray) {
+    @Override
+    override public fun globalSchemeChange(scheme: EditorColorsScheme?) {
+        System.out.println("test");
+        if (scheme?.name=="Darcula" || scheme?.name=="High contrast") {
+            uiTheme = "dark"
+
+        } else {
+            uiTheme = "kennedy"
+        }
```

Comment:
This is duplicated code!
Maybe you can use `PropertyView<T>` to abstract this!

Reply:
I guess I will learn during my Kotlin training how to do this :-)